day6/day6.rb

```diff
@@ -0,0 +1,71 @@
+require_relative './input'
+
+class Day6
+  def initialize(input)
+
+    @competition = Competition.new(input)
+  end
+
+  def part_one
+    @competition.number_of_wins_product
+  end
+
+  def part_two
+    @competition.number_of_wins_product
+  end
+end
+
+ class Competition
+  def initialize(input)
+
+    document = input.split("\n").reject{ |line| line == ""}
```

Comment:
I think you can do this with ` document = input.split("\n").reject(&:blank?)` as well. Not 100% if `#blank?` is built into ruby or if its an extension added by ActiveSupport. I always forget.